RMSD/rmsd.cpp

```diff
@@ -0,0 +1,174 @@
+#include <stdio.h>
+#include <mpi.h>
+#include <stdlib.h>
+#include <iostream>
+#include <fstream>
+#include <math.h>
+#include <qcprot.h>
+#include <sys/time.h>
+
+using namespace std;
+
+double rmsd(double** xref){
+
+    double **xmobile;
+    double temp;
+    double *pointer = new double[9];
+
+    xmobile = new double*[3];
+    
+    for (int i=0;i<3;i++){
+        xmobile[i] = new double[146];
+        for (int j=0;j<146;j++){
+            temp = rand();
+            xmobile[i][j] = (temp/RAND_MAX)*15;
+        }
+    }
+
+    temp = CalcRMSDRotationalMatrix(xref,xmobile,146,pointer,NULL);
+
+    return temp;
+};
+
+double *block_rmsd(double **xref0, int start, int stop,int step,int rank){
+
+    int bsize;
+    double *results;
+
+    double *t_comp;
+
+    bsize = stop - start;
+
+    results = new double[stop-start];
+    for (int i=0;i<stop-start;i+=step){
+        results[i] = rmsd(xref0);
+    }
+
+    return results;
+
+};
+
+
+int main (){
+    int rank;
+    int ierr;
+    int size;
+//
+//  Initialize MPI.
+//
+    ierr = MPI_Init ( NULL, NULL );
+//
+//  Get the number of processes.
+//
+    ierr = MPI_Comm_size ( MPI_COMM_WORLD, &size );
+//
+//  Get the individual process ID.
+//
+    ierr = MPI_Comm_rank ( MPI_COMM_WORLD, &rank );
+
```

Comment:
You add three lines of comments to the most basic MPI methods, which are not really needed - but don't check error codes... :P